frontend/src/common/constansts/time.ts

```diff
@@ -1,6 +1,4 @@
 export const Time = {
     NOTIFICATION_DURATION: 4000,
     CURRENT_YEAR: new Date().getFullYear(),
-    START_DATE: new Date(`${new Date().getFullYear()}-01-01`),
-    END_DATE: new Date(`${new Date().getFullYear()}-12-31`)
```

Comment:
Don't use it here. It is a potential bug. If user opens app at 31/12/2024 23:59 but after 2 minutes goes to home page, these dates will be old.